src/pages/Hwajong/Login/Login.js

```diff
@@ -1,11 +1,77 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
-const Login = () => {
-    return (
-        <div>
-            
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleIdInput = event => {
+    setEmail(event.target.value);
+  };
+
+  const handlePasswordInput = event => {
+    setPassword(event.target.value);
+  };
+
+  const [isActive, setIsActive] = useState(false);
+  const isSucceedLogin = () => {
+    return email.includes('@') && password.length > 4
+      ? setIsActive(true)
+      : setIsActive(false);
+  };
+
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    if (isActive === true) {
```

Comment:
isActive 자체가 boolean type이기 때문에 true/false 값과 비교할 필요 없음
```suggestion
    if (isActive) {
```